opis-manager/src/assets/config.json.dist

```diff
@@ -1,4 +1,4 @@
 {
   "apiUrl": "http://localhost/OPIS-Manager/API/public/index.php/api/",
-  "years": ["2013/2014", "2014/2015", "2015/2016", "2016/2017", "2017/2018"]
+  "years": ["2013/2014", "2014/2015", "2015/2016", "2016/2017", "2017/2018", "2018/2019"]
```

Comment:
Good, we need to make this automatic (in future), maybe we will get the available "years" from the database.